Doppler.Sap/Mappers/Billing/IBillingMapper.cs

```diff
@@ -5,7 +5,7 @@ namespace Doppler.Sap.Mappers.Billing
     public interface IBillingMapper
     {
         bool CanMapSapSystem(string sapSystem);
-
         SapSaleOrderModel MapDopplerBillingRequestToSapSaleOrder(BillingRequest billingRequest);
+        SapIncomingPaymentModel MapSapIncomingPayment(int docEntry, string cardCode, double docTotal);
```

Comment:
In general, it is better using _decimal_ form money.

```suggestion
        SapIncomingPaymentModel MapSapIncomingPayment(int docEntry, string cardCode, decimal docTotal);
```

Reply:
good. I will update it.

Reply:
Done